### ISSUE TITLE
Increase focus ring contrast in Grid cell focus example

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#5763]
The Grid [cell focus example](https://vaadin.com/docs/latest/components/grid#cell-focus) uses a focus ring color with insufficient contrast. Change the color to conform to the 3:1 minimum non-text contrast in WCAG.
